data/libs/NameGen.lua

```diff
@@ -328,6 +329,10 @@ NameGen.orbitalStarportFormats = {
 	"{name} Outpost",
 	"{name} Citadel",
 	"{name} Platform",
+	"{name} Ring",
+	"{name} Residance",
```

Comment:
Bit of a typo here...

```suggestion
	"{name} Residence",
```

Reply:
Oops. Got confused because of my mother language :P